src/searchForm/autoComplete/overlay/AutoCompleteOverlay.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import AutoComplete, { AutoCompleteProps } from 'autoComplete'
+import Icon from 'icon/itineraryLocation'
+
+export const AutoCompleteOverlay = (props: AutoCompleteProps) => {
+  const icon = (
+    <div className="kirk-autoComplete-icon">
```

Comment:
Why it isn't already handled in the `AutoComplete` components for all the `inputAddon`?

Reply:
Since we want to keep retro-compatibility with current version of the AutoComplete component the inputAddon is directly managed by the AutoCompleteOverlay for now